recipes/cnpy/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+import os
+import glob
+
+from conans import ConanFile, CMake, tools
+
+class CnpyConan(ConanFile):
+    name = "cnpy"
+    description = "library to read/write .npy and .npz files in C/C++"
+    license = "MIT"
+    topics = ("conan", "cnpy")
+    homepage = "https://github.com/hongyx11/cnpy"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    _cmake = None
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
```

Comment:
seems to be a C++ project https://github.com/rogersce/cnpy/blob/4e8810b1a8637695171ed346ce68f6984e585ef4/CMakeLists.txt#L8

```suggestion
```